.github/workflows/deploy.yaml

```
@@ -30,15 +36,90 @@ jobs:
+        if [ "${{ env.STACK_NOT_FOUND }}" != "true" ]; then
+          echo "Fetching resources for stack: ${{ env.StackName }}"
+          aws cloudformation describe-stack-resources --stack-name ${{ env.StackName }} \
+            --query "StackResources[*].LogicalResourceId" --output json > resources.json
+          cat resources.json
+        else
+          echo "No resources to track since stack does not exist."
+        fi
+
+    - name: Use manual input for resources to delete
+      id: set_delete_resources
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:80: Double quote to prevent globbing and word splitting

(shellcheck)